iotedgehubdev/cli.py

```diff
@@ -205,7 +206,16 @@ def start(inputs, port, deployment, verbose):
         try:
             with open(deployment) as json_file:
                 json_data = json.load(json_file)
-            edgeManager.start_solution(json_data, verbose)
+                if 'modulesContent' in json_data:
+                    module_content = json_data['modulesContent']
+                elif 'moduleContent' in json_data:
+                    module_content = json_data['moduleContent']
+            try:
+                EdgeManager.login_registries(module_content)
+            except RegistriesLoginError as e:
+                output.info('Warn: {0}'.format(e.message()))
```

Comment:
`Warn` or `Error`?

Reply:
Warn, even login error. The credential may not be used in the modules. So warn at this stage.